sdk/python/feast/infra/utils/hbase_utils.py

```diff
@@ -0,0 +1,189 @@
+from typing import List
+
+from happybase import Connection
+
+from feast.infra.key_encoding_utils import serialize_entity_key
+from feast.protos.feast.types.EntityKey_pb2 import EntityKey
+
+
+class HbaseConstants:
+    """Constants to be used by the Hbase Online Store."""
+
+    DEFAULT_COLUMN_FAMILY = "default"
+    EVENT_TS = "event_ts"
+    CREATED_TS = "created_ts"
+    DEFAULT_EVENT_TS = DEFAULT_COLUMN_FAMILY + ":" + EVENT_TS
+    DEFAULT_CREATED_TS = DEFAULT_COLUMN_FAMILY + ":" + CREATED_TS
+
+    @staticmethod
+    def get_feature_from_col(col):
+        """Given the column name, exclude the column family to get the feature name."""
+        return col.decode("utf-8").split(":")[1]
+
+    @staticmethod
+    def get_col_from_feature(feature):
+        """Given the feature name, add the column family to get the column name."""
+        if isinstance(feature, bytes):
+            feature = feature.decode("utf-8")
+        return HbaseConstants.DEFAULT_COLUMN_FAMILY + ":" + feature
```

Comment:
What does a column family represent here? Is there a reason to use the default one for all features? Would that result in squashing, if multiple feature views had the same feature name?

Reply:
In HBase, every column should belong to a column family. In our use case, we use featre names as column, thus we add a default column family for all the features.

**If multiple feature views had the same feature name?**

Please note that every feature view creates a different table in HBase.  We use <project_name>_<feature_view> as the table name in HBase. Thus, there is no chance of collision even if multiple feature views had the same feature name.

Reply:
Got it, that addresses my concern.